src/Implement/MainManager.cpp

```diff
@@ -0,0 +1,107 @@
+#include "MainManager.h"
+
+
+namespace{
+
+    MainManager* onlyMainManager = nullptr;
+}
+
+void SigHandle(int sig, siginfo_t *info, void *ctx){
+
+    if(sig == SIGUSR1)
+    {
```

Comment:
Please put curly brackets and newline with consistency.

Reply:
Done

Reply:
This review is reasonable. Furthermore, I would suggest doing clang-format.

Reply:
Done